addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14436,7 +14436,7 @@ msgstr ""
 #. Description of setting "Videos -> Playback -> Activate Teletext" with label #23050
 #: system/settings/settings.xml
 msgctxt "#36174"
-msgid "Enable Teletext when watching a live TV stream."
+msgid "Enable teletext when watching a live TV stream."
```

Comment:
@opdenkamp says this **teletext** is correct so now has been reverted to teletext.